.infra/common.ts

```diff
@@ -297,6 +297,18 @@ export const workers: Worker[] = [
     topic: 'api.v1.post-visible',
     subscription: 'api.post-added-user-notification',
   },
+  {
+    topic: 'api.v1.source-post-moderation-submitted',
+    subscription: 'api.v1.source-post-moderation-submitted-notification',
+  },
+  {
+    topic: 'api.v1.source-post-moderation-approved',
+    subscription: 'api.v1.source-post-moderation-approved-notification',
+  },
+  {
+    topic: 'api.v1.source-post-moderation-rejected',
+    subscription: 'api.v1.source-post-moderation-rejected-notification',
```

Comment:
usually subscriptions are without `v1`
```suggestion
    subscription: 'api.source-post-moderation-rejected-notification',
```

also above lines